contracts/stake-cw20-rewards/src/contract.rs

```diff
@@ -161,5 +163,486 @@ pub fn query_config(deps: Deps) -> StdResult<GetConfigResponse> {
     })
 }
 
+pub fn query_rewards(deps: Deps, env: Env) -> StdResult<GetPendingRewardsResponse> {
```

Comment:
What do you think about renaming this to `query_pending_rewards`?

Reply:
I agree this needs to be renamed.